### PR TITLE
Add First and Last buttons to the demo

### LINKED ISSUE #52: Pagination, wrong height and incorrect background color on disabled
- The height should be 32px.
- Transparent text backgrounds.
- The devider line should be 1px with and 20px high, vertical centerd

image

### PR DESCRIPTION
**Pull request by alshakero**
And change spans to anchors to fix https://github.com/Starcounter/uniform.css/issues/113.

Fixes #52

### CODE REVIEW

**Comment by tomalec** [comment]
What is it supposed to fix from #52? as margins look still uneven to me 
image

File: index.html [hunk 1]
```diff
@@ -541,36 +541,48 @@ <h2 id="pagination">Pagination</h2>
...
-                </template>
-                <span slot="foo/previous" onclick="doYourJobInJS()" href="orUseStaticLinks">Previous</span>
-                <span>2</span>
-                <span aria-current="true">3</span>
-                <span aria-current="false">4</span>
-                <span disabled>5</span>
-                <span slot="foo/next" disabled>Next</span>
-            </div>
+            <pre><code>                
+&#x3C;div data-shadow-host&#x3E;
```
Comment (tomalec) [inline index.html]: Out of curiosity why do you prefer `&#x3C;`  over `&lt;`?

Comment (tomalec) [inline index.html]: Also, I'd put `<div data-shadow-host>` into code sample

File: index.html [hunk 2]
```diff
@@ -541,36 +541,48 @@ <h2 id="pagination">Pagination</h2>
...
+&#x3C;template is=&#x22;declarative-shadow-dom&#x22; mode=&#x22;open&#x22;&#x3E;
```
Comment (tomalec) [inline index.html]: Other samples uses `<shadowroot>` I'd keep it consistent within at least this page.

File: index.html (hunk 1 above)
Comment (alshakero) [reply index.html]: It's the tool. Not me :D

**Comment by alshakero** [comment]
> What is it supposed to fix from #52? as margins look still uneven to me

well yes, they're indeed uneven. But I thought they were even worse in the issue. I pointed out that I didn't fix them but I think they were fixed before. 

I don't think it's possible to have even margins given the grey border. How would you go about it? I tried many ways and I think if you don't use a seperate `seperator` element we can't make margins even and consider grey bg when its disabled too.

**Comment by alshakero** [comment]
I think we should drop this in favor of https://github.com/Starcounter/uniform.css/issues/97. I designed it in a view that fixes the root cause of this issue.

**Comment by warpech** [comment]
#97 should just be a custom element that implements the logic part.

The look and feel should be identical in `<uni-pagination>` and `.uni-pagination`.

**Comment by alshakero** [comment]
Closing in favour of https://github.com/Starcounter/uniform.css/pull/117

**Pull request closed without merging**